store/store_test.go

```diff
@@ -1986,3 +1986,110 @@ func (t *remoteRepoTestSuite) TestUbuntuStoreBuyFailArgumentChecking(c *C) {
 	c.Assert(err, NotNil)
 	c.Check(err.Error(), Equals, "cannot buy snap \"snap name\": authentication credentials missing")
 }
+
+const paymentMethodsJson = `
```

Comment:
A small comment (just like we do for e.g. `MockUpdatesJSON`) how it was obtained (via curl or httpie) would be nice (or if that is not possible a small comment why not and how it was gathered instead).